django/db/models/functions/math.py

```diff
@@ -0,0 +1,158 @@
+import math
+
+from django.db.models import FloatField, Func, Transform
+
+
+class Abs(Transform):
+    function = 'ABS'
+    lookup_name = 'abs'
+
+
+class ACos(Transform):
+    function = 'ACOS'
+    lookup_name = 'acos'
+    output_field = FloatField()
+
+
+class ASin(Transform):
+    function = 'ASIN'
+    lookup_name = 'asin'
+    output_field = FloatField()
+
+
+class ATan(Transform):
+    function = 'ATAN'
+    lookup_name = 'atan'
+    output_field = FloatField()
+
+
+class ATan2(Func):
+    function = 'ATAN2'
+    arity = 2
+    output_field = FloatField()
+
+
+class Ceil(Transform):
+    function = 'CEILING'
+    lookup_name = 'ceil'
+
+    def as_oracle(self, compiler, connection):
+        return super().as_sql(compiler, connection, function='CEIL')
+
+
+class Cos(Transform):
+    function = 'COS'
+    lookup_name = 'cos'
+    output_field = FloatField()
+
+
+class Cot(Transform):
+    function = 'COT'
+    lookup_name = 'cot'
+    output_field = FloatField()
+
+    def as_oracle(self, compiler, connection):
+        return super().as_sql(compiler, connection, template='1 / TAN(%(expressions)s)')
```

Comment:
Should the content of the template be wrapped in parentheses? (To avoid issues when functions are nested...)

What do you think, @felixxm?